src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceExporterOptions.cs

```diff
@@ -62,5 +63,10 @@ public class DynatraceExporterOptions
 		/// Gets or sets the metric export interval in milliseconds. The default value is 1 minute (60000 ms).
 		/// </summary>
 		public int MetricExportIntervalMilliseconds { get; set; } = 60000;
+
+		/// <summary>
+		/// Gets or sets the <see cref="HttpClient"/> that is used by the exporter
+		/// </summary>
+		internal HttpClient Client { get; set; } = null;
```

Comment:
Do we use this HttpClient for something?

Reply:
No we do not, thanks for catching this. slightly_smiling_face: I was trying to see if can test our `MeterProviderBuilder` extensions, by allowing an internal option for the `HttpClient`. This way we could inject an mocked instance. I removed this in [ed3a0a0](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/31/commits/ed3a0a056078f7916728a092209329fb9b44fa65), but I'm still wondering if that approach might be a valid one, as the extensions are currently untested. 

WDYT? :thinking:

Reply:
Hum I have to see and think about it a bit, but for now I think we can go ahead like this. We can try coming up with something next.